game-core/src/main/resources/META-INF/triplea/product.properties

```diff
@@ -1 +1 @@
-version = 1.10.@buildId@
+version = 1.11.@buildId@
```

Comment:
Errm this isn't going to fix the problem I mentioned
Because we consider 1.10 to be still in development, we had breaking changes all over the place without having to increment the version number each time. So this isn't necessary.
What I wanted to express is that we should avoid Exception messages in the users face that just indicate that they are using the wrong version.

Reply:
Maybe a better way to understand the concern here is to try loading a 1.9 save game and observing the difference in behavior between this branch and `master`.

Reply:
Ya, I did not think there was any expectation for compatibility as well. See the other response for results when checking how the game handles the incompatibility scenario.

Reply:
tl;dr, moving the package of `Version.java` has the same effect as incrementing the version number: 1ffa2a8 added to keep same version number as we get the same effect anyways.

The testing of error handling seems to indicate the way to address this https://github.com/triplea-game/triplea/pull/4690#discussion_r257510742, the concern over handling, is to increment version number; keeping `Version.java` in same package arguably makes it worse as the game then 'thinks' it can load the save-game to then be doomed to fail.

Reply:
@ssoloff I checked the difference of moving `Version.java` from one package to another and it should be noted the exception message/type does not matter currently in the error display. Please see results mentioned in: https://github.com/triplea-game/triplea/pull/4690#discussion_r257515187